azure-bootstrap.yml

```
@@ -0,0 +1,169 @@
+              echo "Failed to get pipeline run status. HTTP Status Code: $http_code"
+              echo "Response: $content"
+              exit 1
+          fi
+
+          # Wait for a while before checking again
+          sleep 30
+      done
+    displayName: 'Monitoring Azure DevOps pipeline'
+    env:
+      run_id: $(run_id)
+      AZURE_DEVOPS_PAT: $(AZURE_DEVOPS_PAT)
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)